app/controllers/charges_controller.rb

```diff
@@ -0,0 +1,38 @@
+require 'stripe'
+class ChargesController < ApplicationController
+  before_action :authenticate_user!
+  before_action :find_story
+  
+  def new
+    
+  end
+  
+  def create
+    @amount = @story.price.to_i * 100
+    
+    customer = Stripe::Customer.create(
+      email: current_user.email,
+      card: params[:stripeToken]
+    )
+    
+    charge = Stripe::Charge.create(
+      customer: customer.id,
+      amount: 	@amount,
+      description: "Purchase of #{@story.title} for #{current_user.email}",
+      currency: 'usd'
+    )
+    
+    #current_user.update(customer_id: customer.id)
```

Comment:
remove commented out stuff if not using for now, u will  have in git history if u ned it back